client/components/credit-card-form-fields/index.jsx

```diff
@@ -90,7 +92,13 @@ export class CreditCardFormFields extends React.Component {
 	};
 
 	shouldRenderEbanxFields() {
-		return shouldRenderAdditionalEbanxFields( this.getFieldValue( 'country' ) );
+		// The add/update card endpoints do not yet process Ebanx payment details
+		// so we only show Ebanx fields at checkout,
+		// i.e., when there is a current transaction.
+		return (
+			get( this.props.transaction, 'step', null ) &&
```

Comment:
Is there a specific reason that we are checking for `step` field here?
From the unit test below, and how we passes in `transaction` prop, it looks like we only need to check `this.props.transaction` is not null here.

Either way, if `CreditCardFormFields` only cares about if there is a transaction going, not about the encapsulated details, I'd also propose we simplify the prop as something like `isNewTransaction` so no further data deriving is needed within this component.

Reply:
That's a good point. When I was looking at the transaction object, step was the most specific property I could find that would definitely tell us that we were in the midst of a transaction.

It's down my personal level of paranoia and not a hard requirement. I'll update it with your suggestion 👍